.gitignore

```diff
@@ -7,3 +7,6 @@ venv/
 .venv/
 .pytest_cache/
 **__pycache__/
+db.sqlite3
```

Comment:
Including `db.sqlite3` in the `.gitignore` file is common practice to avoid committing local database files to the repository. Ensure this is intentional and that no important data is lost by ignoring this file.